recipes/tinycthreadpool/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from conan import ConanFile
+from conans import CMake, tools
+from conan.tools.files import get
+
+required_conan_version = ">=1.53.0"
+
+#
+# INFO: Please, remove all comments before pushing your PR!
+#
+
+
+class PackageConan(ConanFile):
+    name = "tinycthreadpool"
+    version = "1.0"
+    description = "Portable C thread pool"
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case not listed there, use "LicenseRef-<license-file-name>"
+    license = "https://github.com/bennyhuo/tinycthreadpool/blob/master/LICENSE"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bennyhuo/tinycthreadpool"
+    # no "conan" and project name in topics. Use topics from the upstream listed on GH
+    topics = ("C", "Thread Pool", "tinycthread")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake"
+    
+    # no exports_sources attribute, but export_sources(self) method instead
+    # this allows finer grain exportation of patches per version
+    def export_sources(self):
+        self.copy(os.path.join("src/*"))
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        # for plain C projects only
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder="src")
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="src")
+        self.copy("*.lib", dst="lib", keep_path=False)
+        self.copy("*.dll", dst="bin", keep_path=False)
+        self.copy("*.dylib*", dst="lib", keep_path=False)
+        self.copy("*.so", dst="lib", keep_path=False)
+        self.copy("*.a", dst="lib", keep_path=False)
+        self.copy("LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
```suggestion
        self.cpp_info.libs = ["tinycthreadpool"]
```
Avoid collect_libs, it's fragile and does not respect link order. Also, keeping it explicit is much easier than deduce what library name will be available.